deploy/conf/nginx-local.conf

```diff
@@ -0,0 +1,13 @@
+user  nginx;
```

Comment:
Moved here to be used by all experiments from here.

Reply:
I think these live in Faasm, not the experiments repo right? I can't remember what we agreed about this configuration, did we say that it would live in here and we'd delete it from Faasm? 

My gut feeling now would be that we still have the docs on how to deploy Faasm using knative in Faasm itself, and I think that's probably the right place for it. Could we instead have Faasm as a submodule of this repo, then use the existing Faasm tasks? 

Having a Faasm submodule would avoid having to hard-code a `FAASM_VERSION` too (as we could just `cat` the `VERSION` file in the submodule).

Regardless, I think the nginx file here has to live in Faasm as it's used by the quick-start config, and the redis volatile conf is used by the faasm redis container build.

Let's discuss this in person rather than on here, as I might have forgotten some of our previous chats on the topic too.